.github/workflows/ci-windows.yml

```diff
@@ -1,11 +1,14 @@
 name: Windows tests
 
 on:
+  push:
```

Comment:
I'm going to hold off the merge of this PR until https://github.com/testcontainers/testcontainers-go/pull/2737 is completed. This PR is my branch to go fixing the failing tests on Windows